src/services/GiantSquid/CallParser.ts

```diff
@@ -69,3 +69,32 @@ export class WithdrawFromUnbondedParser extends CallParser implements ICallParse
         return result;
     }
 }
+
+// Parses Reward events from a batch call and calculates total rewards per batch.
+@injectable()
+export class BatchCallParser extends CallParser implements ICallParser {
+    public parse(call: DappStakingCallData): UserEvent {
+        const EVENT_NAME = 'Reward';
+        const PALLET_NAME = 'DappsStaking';
+
+        let reward: bigint = BigInt(0);
```

Comment:
`Type bigint trivially inferred from a bigint literal, remove type annotation`
eslint suggest it should be just: `let reward = BigInt(0);`